src/ansys/dpf/core/operators/utility/default_value.py

```diff
@@ -55,41 +59,57 @@ class default_value(Operator):
     _inputs: InputsDefaultValue
     _outputs: OutputsDefaultValue
 
-    def __init__(self, forced_value=None, default_value=None, config=None, server=None):
+    def __init__(
+        self,
+        primary_value=None,
+        default_value=None,
+        config=None,
+        server=None,
+        forced_value=None,
+    ):
         super().__init__(name="default_value", config=config, server=server)
         self._inputs = InputsDefaultValue(self)
         self._outputs = OutputsDefaultValue(self)
-        if forced_value is not None:
-            self.inputs.forced_value.connect(forced_value)
+        if primary_value is not None:
+            self.inputs.primary_value.connect(primary_value)
+        elif forced_value is not None:
+            warn(
+                DeprecationWarning(
+                    f'Operator default_value: Input name "forced_value" is deprecated in favor of "primary_value".'
```

Comment:
@cbellot000 was this change intentional? Forced value makes more sense to me

Reply:
I thought it makes sense but it's not strongly required. I can revert if needed @rafacanton